pipelines/batch/pom.xml

```diff
@@ -77,6 +77,12 @@
       <groupId>mysql</groupId>
       <artifactId>mysql-connector-java</artifactId>
       <version>${mysql.driver.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.protobuf</groupId>
```

Comment:
nit: Please consider adding a comment why this is needed (and what it clashes with).

Reply:
Done